templates/data/incubator-buildpack-data.yaml

```diff
@@ -257,14 +261,18 @@ templates:
               pipeline.clusterprovisioning: kyma cli
           reconciler_nightly_cluster_jobConfig:
             privileged: false
-            env:
-              INPUT_CLUSTER_NAME: "rec-night"
             command: "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/cluster-integration/reconciler-gardener-long-lasting.sh"
             annotations:
               pipeline.platform: "gardener_gcp"
               pipeline.installer: reconciler
               pipeline.test: "fast-integration"
               pipeline.clusterprovisioning: reconciler
+          reconciler_main_nightly_cluster_input_name:
+            env:
+              INPUT_CLUSTER_NAME: "rec-main-night"
```

Comment:
The cluster name is used as a part of the DNS that Gardener provides, and it is used in some of Kubernetes object names. That's why the name should be really short, otherwise we hit into problems with size limits of the Kubernetes object names. I'd suggest naming this cluster `rec-main` and the other one `rec-dev` to avoid this problem.